cmd/qsctl/init.go

```diff
@@ -122,13 +123,20 @@ func initConfig() (err error) {
 
 	// if env not set, start interactive setup
 	if viper.GetString(constants.ConfigAccessKeyID) == "" && viper.GetString(constants.ConfigSecretAccessKey) == "" {
+		// if not run interactively, return error
+		if !utils.IsInteractiveEnable() {
+			log.Errorf("qsctl not run interactively, and cannot load config with err: [%v]", err)
+			return err
```

Comment:
`err` maybe `nil` here.

Reply:
If `err` was `nil`, it would return at line 116.
```
err = viper.ReadInConfig()
if err == nil {
	log.Debugf("Load config success from [%s]: %v", viper.ConfigFileUsed(), viper.AllSettings())
	return
}
```